src/main.rs

```diff
@@ -69,6 +69,8 @@ Options:
     -d, --dot <file>    write out GraphViz representations of the input file(s).
     --edit <file>       write a GraphViz representation of the mapping store to
                         file, after the edit script has been generated.
+    --grammar           use the 'debug' version of the relevant grammar, if
```

Comment:
This seems like a weird name, and weird thing, for the option to do. Why would I want a debug grammar? I suspect I'm missing out on something!